### PR TITLE
New package: durdraw-0.15.1

### PR DESCRIPTION
**Pull request by Barbaross93**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES** ?

#### Local build testing
- I built this PR locally for my native architecture, x86-64

Admittedly, this may not meet the quality requirements, but as far as I know, there are no ANSI art editors in the repos. So this PR can help fill that niche.

### CODE REVIEW

File: srcpkgs/durdraw/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'durdraw'
+pkgname=durdraw
+version=0.15.1
+revision=1
+build_style=meta
```
Comment (classabbyamp) [inline srcpkgs/durdraw/template:5]: > packages that only install local files or simply depend on additional packages. This build style does not install dependencies to the root directory

this is not a metapackage


```suggestion
```

File: srcpkgs/durdraw/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+depends="python3 python3-Pillow"
+short_desc="Animated Unicode, ANSI and ASCII Art Editor for Linux"
+maintainer="Barbaross <cullen.ross@proton.me>"
+license="ISC"
+homepage="https://github.com/cmang/durdraw"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/durdraw/template:11]: expand $homepage here

File: srcpkgs/durdraw/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+checksum=99bd0f14a906a7918f4a1431587814a34d1016c6d2b51e6b7837c9e0b7af27d8
+
+do_install() {
+	vbin durdraw
+}
+
+post_install() {
+	vlicense LICENSE
```
Comment (classabbyamp) [inline srcpkgs/durdraw/template:19]: just do this in do_install, no reason to separate it

**Comment by cmang** [comment]
[Durdraw](https://github.com/cmang/durdraw) has had many updates and gained many more users since this issue was closed. Any thoughts on revisiting adding this package?

Thanks!

**Comment by Barbaross93** [comment]
I've been keeping a local package up to date on my machine. I just haven't updated those changes here since I didn't think the void team was interested in including durdraw in the repos. @classabbyamp if I'm wrong about this, let me know and I'll happily update this PR asap.

**Pull request closed without merging**